src/components/Filter/Filter.js

```diff
@@ -4,11 +4,9 @@ import PropTypes from 'prop-types'
 
 import styles from './Filter.scss'
 
-const Filter = ({ detail, remove, filter, style, onClick, t, i18n }) => {
+const Filter = ({ detail, remove, filter, style, onClick, t }) => {
   const title = t(`components.Filter.${filter.name}`)
-  const value = i18n.exists(`components.Filter.${filter.value}`)
-    ? t(`components.Filter.${filter.value}`)
-    : filter.value
+  const value = t([`components.Filter.${filter.value}`, filter.value])
```

Comment:
This can be dangerous if there's a matching translation string in the global scope though.

Reply:
So the risk would be that the filter be …translated?

Reply:
Yes, maybe using `exists` makes more sense here.